Creational/README.rst

```diff
@@ -6,7 +7,7 @@ suitable to the situation. The basic form of object creation could result in
 design problems or added complexity to the design. Creational design patterns
 solve this problem by somehow controlling this object creation.
 
-* [AbstractFactory](AbstractFactory) [:notebook:](http://en.wikipedia.org/wiki/Abstract_factory_pattern)
+* :doc:`AbstractFactory <AbstractFactory/README>` `Wikipedia <http://en.wikipedia.org/wiki/Abstract_factory_pattern>`_
```

Comment:
1. Is an **Internal** link

   From the documentation http://sphinx-doc.org/markup/inline.html#cross-referencing-documents 

   > Cross-referencing documents
   > There is also a way to directly link to documents:
   > :doc:
   > Link to the specified document; the document name can be specified in absolute or relative fashion. For example, if the reference :doc:`parrot` occurs in the document sketches/index, then the link refers to sketches/parrot. If the reference is :doc:`/people` or :doc:`../people`, the link refers to people.

   Therefore, an extension is not require - I read somewhere it would work for PDFs too.

2. Is an **External** link